utils/docker/cc-builder/Dockerfile

```diff
@@ -4,6 +4,9 @@
 
 FROM hyperledger/fabric-private-chaincode-dev:latest
 
+ARG SGX_MODE=SIM
```

Comment:
do you need both of these?

Reply:
You mean ARG and ENV. Yes, they have different effects. I wanted it persistent in the container to have it consistent on how the container was built, for which you need the ENV. and the ARG is the only way you can pass something at build ...